### DISCUSSION TITLE
[108600] Project Zomboid [仅本体]

### DISCUSSION THREAD

**Discussion by SCHEN1974** [pjy612/SteamManifestCache#31] [Share]
appid 108600
depotid 108600
内测码：A64BCD9C67AB65B537CE432F337ED266C6D41890
[Project Zomboid.zip](https://github.com/pjy612/SteamManifestCache/files/13681268/Project.Zomboid.zip)

**Comment by pjy612** [comment]
how find you?

**Reply by SCHEN1974** [reply]
邮箱：livexs@vip.qq.com Thanks

**Reply by pjy612** [reply]
查收
